### ISSUE TITLE
XMR strak stuck after restarting

### ISSUE THREAD

**Issue by pimmesz** [fireice-uk/xmr-stak#2109]
After rebooting my system I start to restart the XMR stak miner. It seems to have retained the pool address which I entered before but afterwards nothing happens... Anyone experienced this before and know how I can solve it?

![2018-12-02 16_58_40-c__users_admin_desktop_xmr-stak-win64-2 6 0_xmr-stak exe](https://user-images.githubusercontent.com/7393354/49341876-d28d6380-f653-11e8-9681-8a8bad1957b7.png)

**Comment by Spudz76** [comment]
If running from a Shortcut, make sure the "Start in" is set to the right folder where your config files are (working directory) and not some other place, or blank.

**Comment by Spudz76** [comment]
If your shortcut uses some other (nonexistent now) path... it won't have a place to save the configs, which is your problem.

Windows allows for nonexistent workdirs for some reason, illogical as it is.  Fix the shortcut or run the exe directly from within the install folder (not via shortcut) and make a new shortcut.

**Comment by pimmesz** [comment]
Yes found the pools.txt and was able to change everything by hand. Thank you!
